java/cuvs-java/pom.xml

```diff
@@ -134,6 +136,9 @@
                                 <compileSourceRoot>${project.basedir}/src/main/java22</compileSourceRoot>
                             </compileSourceRoots>
                             <multiReleaseOutput>true</multiReleaseOutput>
+                            <excludes>
+                                <exclude>module-info.java</exclude>
```

Comment:
`module-info.java` is skipped only for the Java22 build.  It's already built as part of the Java21 compile step.

This is to avoid errors during the Java 22 build, where `module-info.class` from Java 21 is clobbered.